mlflow/models/evaluation/default_evaluator.py

```diff
@@ -474,7 +473,9 @@ def _log_image_artifact(
         try:
             pyplot.clf()
             do_plot()
-            pyplot.savefig(artifact_file_local_path)
+            pyplot.xticks(fontsize=10)
+            pyplot.yticks(fontsize=10)
+            pyplot.savefig(artifact_file_local_path, bbox_inches="tight")
```

Comment:
We should not call this here.

Reply:
You can call this right after creating SHAP plots.

Reply:
like this ?
```
            pyplot.clf()
            pyplot.xticks(fontsize=10)
            pyplot.yticks(fontsize=10)
            do_plot()
            pyplot.savefig(artifact_file_local_path, bbox_inches="tight")
```

Reply:
No.

Reply:
I meant in `plot_beeswarm` and `plot_beeswarm`.

Reply:
emm, after apply change https://github.com/mlflow/mlflow/pull/6721#discussion_r964479412 the plot becomes
![shap_beeswarm_plot_on_data_diabetes_dataset](https://user-images.githubusercontent.com/19235986/188817028-47a13250-397d-4851-9ebc-8c9d021b82d0.png)

Reply:
I think this is because the image size is fixed.

Reply:
updated according to your suggestion.

Reply:
I meant you should do:

```
        def plot_beeswarm():
            shap.plots.beeswarm(shap_values, show=False, color_bar=True)
            pyplot.xticks(fontsize=10)
            pyplot.yticks(fontsize=10)
            pyplot.savefig(artifact_file_local_path, bbox_inches="tight")
            _adjust_color_bar()
```

Reply:
yes. that's what current code does.